packages/@react-spectrum/s2/src/Breadcrumbs.tsx

```diff
@@ -67,38 +69,27 @@ const wrapper = style<BreadcrumbsStyleProps>({
   }
 }, getAllowedOverrides());
 
-const BreadcrumbsInternalContext = createContext<BreadcrumbsProps<any> & {length: number}>({length: 0});
-
-function Breadcrumbs<T extends object>({
+function Breadcrumbs<T extends object>(props: BreadcrumbsProps<T>) {
+  let {
     UNSAFE_className = '',
     UNSAFE_style,
     styles,
-    ...props
-}: BreadcrumbsProps<T>) {
-  let {size = 'M', isDisabled} = props;
+    size = 'M',
+    children,
+    ...otherProps
+  } = props;
   let ref = useRef(null);
-  // TODO: Remove when https://github.com/adobe/react-spectrum/pull/6440 is released
-  let childArray: ReactElement[] = [];
-  Children.forEach(props.children, (child, index) => {
-    if (isValidElement<{index: number}>(child)) {
-      child = cloneElement(child, {key: index, index});
-      childArray.push(child);
-    }
-  });
   return (
     <RACBreadcrumbs
-      {...props}
+      {...otherProps}
       ref={ref}
       style={UNSAFE_style}
       className={UNSAFE_className + wrapper({
```

Comment:
I forgot that style macros always returned with a leading space, we should call that out somewhere
also, would it be good if they ended with a trailing space for similar reasons of concatenation?

Reply:
idk, but that's outside the scope of this PR.

Reply:
The trailing space would be nice.

Reply:
agreed, outside scope of this PR